hawkular-tenant-jaxrs-filter/pom.xml

```diff
@@ -64,8 +64,8 @@
 
   <dependencies>
     <dependency>
-      <groupId>javax</groupId>
-      <artifactId>javaee-api</artifactId>
+      <groupId>org.jboss.resteasy</groupId>
```

Comment:
They freaked out for no reason, then. If that helps to standardize on this filter, then so be it. But there's no difference between those two dependencies, technically speaking...